tests/db/test_composites.py

```diff
@@ -51,3 +51,7 @@ def test_eq(self):
             Address(street=str("another street"), state=str("NJ"), zip=str("456")),
         )
         self.assertFalse(Address(street=str("other street"), state=str("NJ"), zip=str("456")).__eq__(None))
+
+    def test_bool(self):
+        self.assertTrue(Address(street=str("street")))
+        self.assertFalse(Address())
```

Comment:
By default all models with composites will initialize composite instance even if all composite columns are null. this at least makes it possible to check if anything is initialized in composite